libmscore/chordlist.cpp

```diff
@@ -617,11 +618,11 @@ bool ParsedChord::parse(const QString& s, const ChordList* cl, bool syntaxOnly,
 #endif
       lastLeadingToken = _tokenList.size();
       // get extension - up to first non-digit other than comma or slash
-      for (j = 0, tok1 = ""; i < len; ++i, ++j) {
+      for (tok1 = ""; i < len; ++i) {
             if (!s[i].isDigit() && s[i] != ',' && s[i] != '/')
                   break;
-            tok1[j] = s[i];
-            }
+            tok1.push_back(s[i]);
+      }
```

Comment:
this is changeing the old 3.x code style. Not a big deal though, more a nitpick...

Reply:
Oops, I've brought the old style back.